pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
   "rioxarray",         # wraps rasterio and xarray. Almost superceded by hydromt/raster.py
   "pandas",            # Dataframes
   "pyflwdir>=0.5.4",   # Hight models and derivatives
+  "pyogrio>=0.6",      # io for geopandas dataframes
```

Comment:
Similar to other optional io packages I would suggest to move pyogrio to the optional io dependencies until it the default engine of geopandas.